Classes/Controller/ModuleController.php

```diff
@@ -385,18 +385,20 @@ public function fileReferenceAction($element = 0): ResponseInterface
         } catch (\Exception) {
             $contentElements = [];
         }
+        $files = [];
         // If we just handled a content element, get related data to display as a confirmation
-        if ((int)$element > 0) {
```

Comment:
rector might have to be reconfigured

Reply:
@DavidBruchmann yes, please have a look at #261